pkg/sqlgen/sqlgen.go

```diff
@@ -251,7 +258,12 @@ func (t *MetaTable) DeleteDML(values []interface{}) (string, error) {
 		if !found {
 			return "", errors.Errorf("DeleteDML couldn't find fieldName %s", fn.String())
 		}
-		val, err := maybeDecodeBase64(values[i], isBase64EncodedFieldType(ft))
+
+		val := values[i]
+		var err error
+		if isBase64EncodedFieldType(ft) {
+			val, err = decodeBase64(val)
+		}
```

Comment:
do you feel strongly about keeping these changes in place in terms of a refactor? seems like a "null" impact change at best?

Reply:
(includes the change to both UpsertDML and DeleteDML)